modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/CommandProcessor.java

```diff
@@ -889,10 +889,30 @@ else if (cmdH2 instanceof GridSqlAlterTableAddColumn) {
                             }
                         }
 
+                        int precision = -1;
+                        int scale = -1;
+
+                        if (col.column().getType().getValueType() == Value.DECIMAL) {
+                            if (col.precision() < H2Utils.DECIMAL_DEFAULT_PRECISION) {
+                                precision = col.precision();
+                            }
+
+                            if (col.scale() < H2Utils.DECIMAL_DEFAULT_SCALE) {
```

Comment:
why you operate with "<" and not with "!=" here ?

Reply:
This code is already in use, I have extracted it into a separate method for clarity.